easybuild/easyconfigs/f/FSL/FSL-6.0.2-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,77 @@
+# This build still relies on the following libraries (or part of them) shipped with FSL:
+#    GDCHART, libprob, libcprob, cprob, newran, newmat
+# Attempts to use externally built libraries failed. Worth trying again in the future...
+#
+# Some tools like icmp are missing since they are built with the fslpython_install.sh script
+# which locally installs Miniconda and creates a virtual environment. This is too messy and
+# an alternative solution is necessary to install them.
+# https://github.com/easybuilders/easybuild-easyconfigs/issues/7899
+# https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FslInstallation#imcp.2Fimglob.2Fimmv_errors_after_installation
+
+name = 'FSL'
+version = '6.0.2'
+versionsuffix = '-Python-3.7.2'
+
+homepage = 'https://www.fmrib.ox.ac.uk/fsl/'
+description = """FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ["https://www.fmrib.ox.ac.uk/fsldownloads/"]
+sources = ['%(namelower)s-%(version)s-sources.tar.gz']
+patches = [
+    'FSL-%(version)s_Makefile_fixes.patch',
+    'FSL-%(version)s_Build_extras.patch',
+    'FSL-6.0.1_Melodic-use-ifstream-good.patch',
+    'FSL-6.0.1_Fix_fsl_exec_script.patch',
+]
+checksums = [
+    'c118b351c6cedb441af7e1b9d194cf344505ff53b417063f697b86305a908afd',  # fsl-6.0.2-sources.tar.gz
+    '4212478ef24be4bce7a9ce513aa9c45fcf67eccfe064331a2e8e52be41d3513c',  # FSL-6.0.2_Makefile_fixes.patch
+    '168157f07818e7dfd04c222916e663e9d21db84b4b86df5b79bab56e3bf8ccf5',  # FSL-6.0.2_Build_extras.patch
+    'c07644fbd89cf9c70db5a1a8f4f2918ab5daeb60cdf0ce4ee2b91f8ae48173fa',  # FSL-6.0.1_Melodic-use-ifstream-good.patch
+    'aa155f8576dc5f010757ecf66fc0bf673454b6c6c40346cbb01cbe59236ed6ef',  # FSL-6.0.1_Fix_fsl_exec_script.patch
+]
+
+builddependencies = [
+    ('Boost', '1.70.0'),
+]
+
+dependencies = [
```

Comment:
add Python dependency

Reply:
I don't think Python is actually a dependency of FSL, but `VTK` needs it.

Reply:
same as above, add it is so you can use the `%(pyver)s` template in the versionsuffix.

Reply:
Well, actually, hardcoding the version dependency in `versionsuffix` makes more sense in this case...
It's a bit misleading to include `Python` as a direct dep if it's not actually a depedency.

Reply:
Done.